<?xml version="1.0"?>
<launch>

    <node pkg="localization" type="localization_node" name="localization_node" output="screen">
    </node>

</launch>
